### PR TITLE
[WIP] Add canned multi label head

### PR DESCRIPTION
**Pull request by yupbank**
Work in Progress.

multi label head was provided in tf.contrib.learn, but not in the canned head_lib

### CODE REVIEW

**Comment by yupbank** [comment]
close this, since the contrib version is finalized

File: tensorflow/python/estimator/canned/head.py [hunk 1]
```diff
@@ -1273,6 +1274,316 @@ def _create_tpu_estimator_spec(
...
+      return metric_ops
+
+  def create_loss(self, features, mode, logits, labels):
+    """See `Head`."""
+    del mode  # Unused for this head.
+    logits = ops.convert_to_tensor(logits)
+    labels = _check_dense_labels_match_logits_and_reshape(
+        labels=labels, logits=logits, expected_labels_dimension=self.logits_dimension)
+    if self._label_vocabulary is not None:
+      sparse_labels = string_ops.string_split(labels, '.')
```
Comment (yupbank) [inline tensorflow/python/estimator/canned/head.py]: `s/'.'/self._label_sep/`
